<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectCatDialog</class>
 <widget class="QDialog" name="selectCatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>اختيار قسم</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>بحث:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="FilterLineEdit" name="lineSearch">
     <property name="placeholderText">
      <string>بحث عن قسم</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="pushCancel">
     <property name="text">
      <string>الغاء</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>279</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushSelect">
     <property name="text">
      <string>اختيار</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>filterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
